src/server/api/boards/boards.js

```diff
@@ -174,6 +189,38 @@ function createNewColumn(request, username, boardId, callback) {
   });
 }
 
+function deleteColumnId(username, boardId, columnsId, callback) {
+  MongoClient.connect(url, function(err, database) {
+    if (err) {
+      return callback('error');
+    }
+    console.log('Connection established to ' + url);
+    let query = columnQuery(username, boardId, columnsId);
+
+    if (!query) {
+      return callback('notFound');
+    }
+    database.collection('boards').findOne(query, function(err, result) {
+      console.log(result);
```

Comment:
remove this or add some description so we don't see random things in the console.
like
```js
console.log(`delete column result: ${result}`);
```